src/Record.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Collection;
+
+readonly class Record
+{
+    public int $id;
+    public string $album_name;
+    public string $artist_name;
+    public int $genre_id;
```

Comment:
Once you've made the changes to the model, you can get rid of this genre_id prop and replace it with the actual genre name itself